tests/sentry/monitors/consumers/test_monitor_consumer.py

```diff
@@ -129,6 +129,10 @@ def check_processing_errors(
             yield
 
             args_list = handle_processing_errors.call_args_list
+            if expected_error.processing_errors == []:
+                assert len(args_list) == 0
+                return
```

Comment:
I don't love this, I'm thinking maybe you could define like

```py
class ExpectNoProcessingError:
  pass
```

and then have 

```py
excpected_error: ProcessingErrorsException | ExpectNoProcessingError | None,
```

and pass like

```py
expected_error=ExpectNoProcessingError(),
```

Reply:
ahhh i see what you're getting at now, tyty just changed